questions.md

```diff
@@ -0,0 +1,8 @@
+#Additional Questions
+
+* Here is where I can list any additional questions that I have
+
+1. I'm wondering what problems I may run into in the classroom
+2. What will I need to train my students on
```

Comment:
At a minimum, I recommend they understand Git Basics, Network Activity, and Collaboration features (Modules 1-3 of this training).